data/net/minecraft/commands/arguments/ArgumentSignatures.mapping

```diff
@@ -3,6 +3,11 @@ CLASS net/minecraft/commands/arguments/ArgumentSignatures
 		ARG 1 buffer
 	METHOD get (Ljava/lang/String;)Lnet/minecraft/network/chat/MessageSignature;
 		ARG 1 key
+	METHOD lambda$signCommand$1 (Lnet/minecraft/commands/arguments/ArgumentSignatures$Signer;Lnet/minecraft/network/chat/SignableCommand$Argument;)Lnet/minecraft/commands/arguments/ArgumentSignatures$Entry;
+		ARG 1 signedArgument
+	METHOD lambda$write$0 (Lnet/minecraft/network/FriendlyByteBuf;Lnet/minecraft/commands/arguments/ArgumentSignatures$Entry;)V
+		ARG 0 byteBuf
```

Comment:
```suggestion
		ARG 0 buffer
```

As per previously established conventions, [here](https://github.com/ParchmentMC/Parchment/pull/207#discussion_r1248014538), and [here](https://github.com/ParchmentMC/Parchment/pull/24#pullrequestreview-695217286)